src/radical/entk/appman/wfprocessor.py

```diff
@@ -100,145 +110,173 @@ def _initialize_workflow(self):
 
         except Exception, ex:
             self._logger.exception(
-                'Fatal error while initializing workflow: %s' % ex)
+                'Fatal error when initializing workflow: %s')
             raise
```

Comment:
`%s` is unresolved, `ex` is not used anymore?

Reply:
No longer exists.

Reply:
Actually, it exists and is fixed.